tools/XRefService/DatabaseLib/DatabaseLib.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Also use the new format csrpoj and reuse `..\..\src\Shared\base.props`, example: https://github.com/dotnet/docfx/blob/dev/tools/AzureMarkdownRewriterTool/AzureMarkdownRewriterTool.csproj